### ISSUE TITLE
Ships manoeuvres

### ISSUE THREAD

**Issue by Saurbaum** [geordanr/xwing#195]
I would be nice to have an option to print the ships available manoeuvres along with all the upgrades on the fancy sheet to make it easier to reference than  digging out the various bits of cardboard.

**Comment by rognstad** [comment]
I agree that this would be nice.

**Comment by xwinger** [comment]
Agreed, especially since it looks like there is already a dial being generated that gets updated with any applicable modifications in the application itself.  Getting that on the print out would be great.

**Comment by Wildhorn** [comment]
I agree this would be nice.
